test/capybara/screenshot/diff/test_methods_test.rb

```
@@ -103,6 +105,10 @@ def test_cleanup_base_image_for_changes
         def our_screenshot(name, skip_stack_frames)
           screenshot(name, skip_stack_frames: skip_stack_frames)
         end
+
+        def assert_image_not_changed(*)
+          CapybaraScreenshotDiff::ScreenshotAssertion.assert_image_not_changed(*)
```

RuboCop (1.73)

[fatal] unexpected token tRPAREN
(Using Ruby 3.1 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)